src/esp/bindings_js/index.js

```diff
@@ -1,12 +1,26 @@
-/*global Module */
+/* global FS, Module */
 
 import WebDemo from "./modules/web_demo";
 import VRDemo from "./modules/vr_demo";
 import "./bindings.css";
 
-Module["onRuntimeInitialized"] = function() {
-  console.log("hsim_bindings initialized");
+Module.preRun.push(() => {
+  let arg;
+  let scene;
+  for (arg of window.location.search.substr(1).split("&")) {
+    let [key, value] = arg.split("=");
```

Comment:
I would add a default value here and make sure that args has length 2 before assigning it to `key, value`.

Reply:
Good idea! That also preserves the prior behavior.